portfolio/src/main/webapp/script.js

```diff
@@ -11,3 +11,25 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+/** Fetches tasks from the server and adds them to the DOM. */
```

Comment:
nit: Update comment to "Fetches comments from ..."

Reply:
Done